docs/03-tutorials/00-eventing/evnt-04-change-max-in-flight-in-sub.md

```diff
@@ -0,0 +1,300 @@
+   ```bash
+   go install github.com/cloudevents/conformance/cmd/cloudevents@latest
+   ```
+
+## Create a Workload
+
+First, create a sample Function that prints out the received event to console:
+
+<div tabs name="Deploy a Function" group="create-workload">
+  <details open>
+  <summary label="Kyma Dashboard">
+  Kyma Dashboard
+  </summary>
+
+1. Go to **Namespaces** and select the default Namespace.
+2. Go to **Workloads** > **Functions** and click **Create Function +**.
+3. Name the Function `lastorder` and click **Create**.
+4. In the inline editor for the Function, replace its source with the following code:
+   ```js
+   module.exports = {
+     main: async function (event, context) {
+       console.log("Processing event:", event.data);
+       // sleep/wait for 5 seconds
+       await new Promise(r => setTimeout(r, 5 * 1000));
+       console.log("Completely processed event:", event.data);
+       return;
+     } 
+   }
+   ```
+   The Function will wait for 5 seconds before returning the response in order to simulate prolonged event processing.
+5. Save your changes.
+6. Wait a few seconds for the Function to have status `RUNNING`.
+
+  </details>
+  <details>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+```bash
+cat <<EOF | kubectl apply -f -
+  apiVersion: serverless.kyma-project.io/v1alpha1
+  kind: Function
+  metadata:
+    labels:
+      serverless.kyma-project.io/build-resources-preset: local-dev
+      serverless.kyma-project.io/function-resources-preset: S
+      serverless.kyma-project.io/replicas-preset: S
+    name: lastorder
+    namespace: default
+  spec:
+    deps: '{ "dependencies": {}}'
+    maxReplicas: 1
+    minReplicas: 1
+    source: |
+      module.exports = {
+        main: async function (event, context) {
+          console.log("Processing event:", event.data);
+          // sleep/wait for 5 seconds
+          await new Promise(r => setTimeout(r, 5 * 1000));
+          console.log("Completely processed event:", event.data);
+          return;
+        } 
+      }
+EOF
+```
+
+The Function will wait for 5 seconds before returning the response in order to simulate prolonged event processing.
+
+<br/>
+If the resources were created successfully, the command returns this message:
+
+```bash
+function.serverless.kyma-project.io/lastorder created
+```
+
+To check the Function status, run:
+
+```bash
+kubectl get functions -n default lastorder
+```
+
+> **NOTE:** You might need to wait a few seconds for the Function to be ready.
+
+  </details>
+</div>
+
+## Create a Subscription with Max-In-Flight config
+
+Next, we will create a [Subscription](../../05-technical-reference/00-custom-resources/evnt-01-subscription.md) custom resource. We're going to subscribe for events of the type: `order.received.v1`. We will set the `maxInFlightMessages` to `5`, so that the Eventing Services forwards maximum 5 events in parallel to the sink without waiting for a response.
```

Comment:
```suggestion
Next, create a [Subscription](../../05-technical-reference/00-custom-resources/evnt-01-subscription.md) custom resource. Subscribe for events of the type: `order.received.v1` and set the `maxInFlightMessages` to `5`, so that the Eventing Services forwards maximum 5 events in parallel to the sink without waiting for a response.
```